### PR TITLE
fix: flash of untranslated title in `settings/my-account/profile`

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

before:
[Screencast 2022-12-22 21:33:54.webm](https://user-images.githubusercontent.com/84864519/209174527-f8250358-2968-4fc5-9e3a-460a504f959f.webm)

after:
[Screencast 2022-12-22 21:33:33.webm](https://user-images.githubusercontent.com/84864519/209174581-f2edd558-46f2-4936-b4e5-76b93e3ef022.webm)

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/ui/v2/core/Meta.tsx [hunk 1]
```diff
@@ -52,9 +52,10 @@ export default function Meta({ title, description, backButton, CTA }: MetaType)
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [title, description, backButton, CTA]);
 
+  const title_ = `${title} | Cal.com`;
```
Comment (G3root) [inline packages/ui/v2/core/Meta.tsx:55]: this change fixed this warning.

`
Warning: A title element received an array with more than 1 element as children. In browsers title Elements can only have Text Nodes as children. If the children being rendered output more than a single text node in aggregate the browser will display markup and comments as text in the title and hydration will likely fail and fall back to client rendering
`
for more context : https://github.com/vercel/next.js/discussions/38256

File: apps/web/pages/settings/my-account/profile.tsx [hunk 2]
```diff
@@ -180,128 +179,135 @@ const ProfileView = () => {
...
-    return <SkeletonLoader title={t("profile")} description={t("profile_description")} />;
+  const isPageLoading = isLoading || !user || isLoadingAvatar || !avatar;
 
   const defaultValues = {
-    username: user.username || "",
-    avatar: avatar.avatar || "",
-    name: user.name || "",
-    email: user.email || "",
-    bio: user.bio || "",
+    username: user?.username || "",
```
Comment (zomars) [inline apps/web/pages/settings/my-account/profile.tsx:185]: By this point user shouldn't be undefined right?

```suggestion
    username: user.username || "",
```

File: packages/ui/v2/core/Meta.tsx (hunk 1 above)
Comment (zomars) [reply packages/ui/v2/core/Meta.tsx:55]: TIL 🙌🏽

Comment (G3root) [reply packages/ui/v2/core/Meta.tsx:55]: New to me also 😀

File: apps/web/pages/settings/my-account/profile.tsx (hunk 2 above)
Comment (G3root) [reply apps/web/pages/settings/my-account/profile.tsx:185]: Yeah you are right but typescript is complaining that the user is `undefined`. Previously the skeleton was rendered before the `Defaultvalue` so it didn't complain. now the loading check is inside the return statement  so it's complaining.

Comment (PeerRich) [reply apps/web/pages/settings/my-account/profile.tsx:185]: has this been resolved? @G3root

Comment (G3root) [reply apps/web/pages/settings/my-account/profile.tsx:185]: @PeerRich i think this is resolved. by the above point the user should be defined but typescript is throwing an error stating that the user is undefined, so i added a `optional chaining` operator additionally.

File: apps/web/pages/settings/my-account/profile.tsx [hunk 3]
```diff
@@ -180,128 +179,135 @@ const ProfileView = () => {
...
-              <Form handleSubmit={onConfirm} className="pb-4" form={form}>
-                <TwoFactor center={false} />
-              </Form>
-            )}
-
-            {hasDeleteErrors && <Alert severity="error" title={deleteErrorMessage} />}
-            <DialogFooter>
+      {isPageLoading ? (
+        <SkeletonLoader />
+      ) : (
```
Comment (zomars) [inline apps/web/pages/settings/my-account/profile.tsx:197]: - This condition should run in a container component
- ProfileForm should receive full `defaultValues` on the first render. If not there will be inconsistencies.

**Review by zomars — APPROVED** [review]
LGTM

**Pull request merged**